packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -3,29 +3,87 @@ import { assert, details } from '@agoric/assert';
 import makeStore from '@agoric/store';
 import makeWeakStore from '@agoric/weak-store';
 import makeAmountMath from '@agoric/ertp/src/amountMath';
+import { makeTable, makeValidateProperties } from '@agoric/zoe/src/table';
```

Comment:
We should at least add a TODO to move the table code to a library outside Zoe, if it's not a good time to move it now.

Reply:
Good idea.